From f577c3618d6e3d086e245f201c3d879922d6f1ca Mon Sep 17 00:00:00 2001
From: "Luis R. Rodriguez" <mcgrof@do-not-panic.com>
Date: Fri, 19 Oct 2012 18:48:37 -0700
Subject: [PATCH] compat-drivers: add target path for ChangeLog file

Upstream commit f577c3618d6e3d086e245f201c3d879922d6f1ca

Forgot to add the target directory for kup upload on the
ChangeLog file.

Signed-off-by: Luis R. Rodriguez <mcgrof@do-not-panic.com>
---
 scripts/gen-release.sh |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/gen-release.sh b/scripts/gen-release.sh
index 6ed4c60..daefca5 100755
--- a/scripts/gen-release.sh
+++ b/scripts/gen-release.sh
@@ -298,7 +298,7 @@ elif [[ "$BASE_TREE" = "linux-stable" ]]; then
 	$STAGING/${RELEASE}/scripts/genlog-${GENLOG_TARGET}
 
 	gpg --armor --detach-sign $CHANGELOG_FILE
-	kup put ${CHANGELOG_FILE} ${CHANGELOG_FILE}.asc
+	kup put ${CHANGELOG_FILE} ${CHANGELOG_FILE}.asc ${TARGET_STABLE}/
 else
 	echo "Unsupported release type: $BASE_TREE"
 	exit 1
-- 
1.7.10.4

